<launch>
 <param name="robot_description"
  command="$(find xacro)/xacro --inorder $(find xpider_descriptions)/models/xpider.xacro" />
 <arg
   name="gui"
   default="True" />
 <node
   name="rviz"
   pkg="rviz"
   type="rviz"
   required="true"
   args="-d $(find xpider_descriptions)/urdf.rviz" />
  <param
    name="use_gui"
    value="$(arg gui)" />
  <param
    name="rate"
    value="25" />

  <node pkg="robot_state_publisher" type="state_publisher" name="robot_state_publisher" required="true">
    <param name="publish_frequency" type="double" value="30.0"/>
    <param name="tf_prefix" type="string" value=""/>
  </node>

  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />

  <node pkg="xpider_descriptions" type="xpider_descriptions" name="xpider_publisher" output="screen" />
<!--  <node pkg="xpider_driver" type="xpider_driver" name="xpider_driver" output="screen" />-->

<!--  <node pkg="tf" type="static_transform_publisher" name="step_connect_1_to_step_b_1" args="1 0 0 0 0 0 1 step_connect_1 step_b_1 100" />-->
</launch>
